### PR TITLE
redirected /fellowship to /fellowships

### PR DESCRIPTION
**Pull request by mmmavis**
As suggested in https://github.com/mozilla/foundation.mozilla.org/pull/996#pullrequestreview-100604752

### CODE REVIEW

File: network-api/networkapi/urls.py [hunk 1]
```diff
@@ -41,14 +41,12 @@
     url(r'^wagtail/', include(wagtail_urls))
     if settings.ENABLE_WAGTAIL else None,
 
+    url(r'^fellowship/(?P<path>.*)', RedirectView.as_view(
+        url='/fellowships/%(path)s',
+        query_string=True
+    )),
```
Comment (mmmavis) [inline network-api/networkapi/urls.py]: not sure if I need to add an extra line in between these 2 lines?

File: network-api/networkapi/urls.py [hunk 2]
```diff
@@ -41,14 +41,12 @@
...
     url(r'^soc/', include('social_django.urls', namespace='social')),
```
Comment (cadecairos) [inline network-api/networkapi/urls.py]: please drop this line too, it's a dupe of line 33

File: network-api/networkapi/urls.py (hunk 1 above)
Comment (cadecairos) [reply network-api/networkapi/urls.py]: nope, not unless pep8 says we need it

Comment (mmmavis) [reply network-api/networkapi/urls.py]: weird, thought i deleted it 👀

File: network-api/networkapi/urls.py (hunk 2 above)
Comment (mmmavis) [reply network-api/networkapi/urls.py]: weird, thought i deleted it 👀

**Pull request merged**